bin/init_repo.sh

```diff
@@ -37,12 +40,21 @@ for template_file in "${template_files[@]}" ; do
         for idx in "${!user_values[@]}" ; do
             placeholder=${placeholders[idx]}
             placeholder_name="${placeholder%%:*}"
-            result=`echo "$result" | sed "s/{{$placeholder_name}}/${user_values[idx]}/g"`
+            result=`echo "$result" | sed "s#{{$placeholder_name}}#${user_values[idx]}#g"`
+            if [ "$placeholder_name" = "SRC" ] && [ ${user_values[idx]} != 'src' ]; then
+              mv "$current_path/../src" "$current_path/../${user_values[idx]}"
+            fi
         done
 
-        echo "$result" > "$dest_path"
-        printf "${green}Processed and created ${nc}$dest_path\n"
-        rm "$source_path"
+        if [ -n "${result}" ];
+        then
+          echo "$result" > "$dest_path"
+          arrPath=(${dest_path//\// })
+          mv $dest_path "${current_path}/../${arrPath[${#arrPath[@]}-1]}"
+          printf "${green}Processed and created ${nc}$dest_path\n"
+        else
+          printf "${red}Empty parsed string. Check your inputs${nc}\n"
```

Comment:
This case should happen only if the original template is empty: is it worth it to add this check?

Reply:
Nope, this happens also when some of the input strings contain the separator character used by `sed` command.
I noticed this while trying to automatically define the github repo URL, I changed it from `/` to `#` to solve that specific conflict but I know no ways to prevent a user to insert a string with `#`, so I added this exception.

Reply:
Ah, ok, then I think (haven't tried and I don't know bash at all) that you can make a replace in the user input by replacing the separator char with an escaped version of itself: `# -> \#`.

Reply:
Of course, you can but the user needs to be aware of the limitation to do that...

Reply:
No, I meant without limitations.
In pseudocode:
```
userInput.replace(/#/g, "\\#")
```

Reply:
So you mean to parse the input value replacing the sed separation character with its escaped version... I wouldn't know how to do that without using `sed` itself...

Reply:
According to [this](https://stackoverflow.com/a/27787551) you can also use control characters as sed separators, so it should be "safe-ish" to assume that the user won't type those.
Otherwise seems nice the solution [here](https://stackoverflow.com/a/63257436) using `printf`.

Reply:
Ok, I found a solution that solves the problem at its root by removing the usage of `sed`

Reply:
Do I smell a hint of `awk` here?